core/src/test/scala/integration/kafka/admin/TopicCommandIntegrationTest.scala

```diff
@@ -697,6 +697,19 @@ class TopicCommandIntegrationTest extends KafkaServerTestHarness with Logging wi
     try {
       killBroker(0)
       killBroker(1)
+
+      val aliveServers = brokers.filterNot(brokers => brokers.config.brokerId == 0 || brokers.config.brokerId == 1)
+
+      if (isKRaftTest()) {
+        TestUtils.ensureConsistentKRaftMetadata(aliveServers, controllerServer,
```

Comment:
We have a helper `KafkaServerTestHarness.ensureConsistentKRaftMetadata`. I think it would make sense to modify it to use only the alive brokers. Then we could use it here.

Reply:
Thank you for the review and suggestion. I have made two changes based on your suggestion:
1. Added a helper method `aliveBrokers` in KafkaServerTestHarness()
2. Modified KafkaServerTestHarness.ensureConsistentKRaftMetadata() to check for only the brokers which are alive

Separately, if you have thoughts on further simplification (especially around combining the logic of Zk and kraft in a single helper function), I would be happy to make the change.